templates/core/terraform/servicebus/variables.tf

```diff
@@ -1,6 +1,6 @@
+variable "tenant_id" {}
```

Comment:
Is `tenand_id` and `resource_processor_subnet_id` used anywhere?

Reply:
`tenant_id` is not and is now removed.

`resource_processor_subnet_id` is used by the *Service Bus* private endpoint - which is a little strange:

```json
resource "azurerm_private_endpoint" "sbpe" {
  ...
  subnet_id           = var.resource_processor_subnet_id
```